### PR TITLE
Rename PParams

### PR DESCRIPTION
**Pull request by WhatisRT**
# Description

Renames the PParams to be in line with the CIP & adds a check that new CC members have their term below the maximum.

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [x] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [x] Self-reviewed the diff

### CODE REVIEW

File: src/Ledger/GovernanceActions.lagda
Comment (WhatisRT) [reply src/Ledger/GovernanceActions.lagda:338]: There's now a proof in `Ledger.Ratify.Properties` called `RATIFY-total` that guarantees that you can always make progress with `RATIFY`. If `ENACT` fails, we delay instead.

**Comment by WhatisRT** [comment]
I've added a tiny commit that also closes #219

File: src/Ledger/Epoch.agda
Comment (williamdemeo) [inline src/Ledger/Epoch.agda:77]: Something like this should have been part of my type classes PR, but okay...

Also, what's the point of `Decidable²⇒Dec`?  Why don't we use `Decidable` directly?
```agda
    Dec-≤ᵉ : ∀ {n m} → Dec (n ≤ᵉ m)
    Dec-≤ᵉ {n}{m} = n ≤ᵉ? m
```

File: src/Ledger/Foreign/HSLedger.agda
Comment (williamdemeo) [inline src/Ledger/Foreign/HSLedger.agda:251]:
```suggestion
        ; govActionDeposit  = govActionDeposit
```

**Review by williamdemeo — CHANGES_REQUESTED** [review]
Just one minor (whitespace/alignment) change requested.  Otherwise, just wondering why we use `Decidable²⇒Dec` instead of just applying `Decidable`.  Maybe you could comment on that...?

**Review by Soupstraw — APPROVED** [review]

File: src/Ledger/Epoch.agda
Comment (WhatisRT) [reply src/Ledger/Epoch.agda:77]: Yes, but your PR is not even out of WIP yet, and I needed this now so that it can be included in the 0.9 release.

I guess `Dec-≤ᵉ` could have been implemented directly, but this highlights a general pattern that we could use in the future.

**Comment by WhatisRT** [comment]
@williamdemeo I've included your suggestion, could you unblock this PR?

File: src/Ledger/Epoch.agda
Comment (williamdemeo) [reply src/Ledger/Epoch.agda:77]: I'll "unblock" it now.

**Review by williamdemeo — APPROVED** [review]

File: src/Ledger/Epoch.agda
Comment (williamdemeo) [reply src/Ledger/Epoch.agda:77]: Sorry, I didn't realize you were waiting for me to approve the change.

**Pull request merged**
